src/text-area-word-count/wordcount-hint-message.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const WordCountHintMessage = ({ content, id, maxWordCount = 0 }) => {
+
+    const wordCount = content?.split(/\s+/).filter(Boolean).length;
+
+    const hintId = `${id}-wordcount-hint`;
+
+    let hintText = '';
+
+    if (!wordCount && maxWordCount) {
+        hintText = `You have ${maxWordCount} words remaining`;
+    }
```

Comment:
The word count 0 case is always overwritten by the else block below (line 20-23). Remove this one and make sure wordCount is not null - this also handles the pluralisation if maxWordCount is ever 1.
```suggestion
    const wordCount = content?.split(/\s+/).filter(Boolean).length ?? 0;

    const hintId = `${id}-wordcount-hint`;

    let hintText = '';
```